source/ConditionsStore.cpp

```diff
@@ -102,13 +102,13 @@ int64_t ConditionsStore::Get(const string &name) const
 
 	// If the name matches exactly, then access directly with explicit conversion to int64_t.
 	if(ce->name == name)
-		return ce->operator int64_t();
+		return *ce;
```

Comment:
Why was this change needed for this PR?

This looks like a change that should have gone into another PR, because I don't see what a conversion to int64 in ConditionsStore has to do with starfield zoom levels.